### ISSUE TITLE
Simplify[Sqrt[x^2], x > 0]

### ISSUE THREAD

**Issue by fja-pedraza** [mathics/Mathics#1595]
This expression should give x as a result, and gives Sqrt[x ^ 2]

**Comment by mmatera** [comment]
This is true. The support for simplification rules is far from being complete right now.
